project_task_project_mandatory/models/project_task.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+# © 2015-2017 Elico corp (https://www.elico-corp.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
+
+
+class ProjectTask(models.Model):
+    _inherit = 'project.task'
+
+    project_id = fields.Many2one(required=True)
```

Comment:
@dongshukai And that is the main problem.

Please study and understand the meaning of parameters.
`required` will try make on DB this `project_id` column not null.

So as we know the column exist so when try alter the table will fail for sure.

Also UT performed by Travis, runbot will fail.